recipes/poco/all/patches/1.10.0.patch

```diff
@@ -0,0 +1,73 @@
+--- cmake/PocoMacros.cmake
++++ cmake/PocoMacros.cmake
+@@ -278,7 +278,7 @@
+ 
+ if (MSVC)
+ # install the targets pdb
+-  POCO_INSTALL_PDB(${target_name})
++  # POCO_INSTALL_PDB(${target_name})
+ endif()
+ 
+ endmacro()
+--- Crypto/CMakeLists.txt
++++ Crypto/CMakeLists.txt
+@@ -21,7 +21,7 @@
+     DEFINE_SYMBOL Crypto_EXPORTS
+     )
+ 
+-target_link_libraries(Crypto PUBLIC Poco::Foundation OpenSSL::SSL OpenSSL::Crypto)
++target_link_libraries(Crypto PUBLIC Poco::Foundation OpenSSL::OpenSSL)
```

Comment:
same, better fix the openssl recipe instead of patching poco

Reply:
funny, the [blog](https://blog.conan.io/2020/05/07/New-conan-release-1-25.html) mentions how to fix the openssl recipes for  compontents, but no one did it, related to https://github.com/conan-io/conan-center-index/issues/952

Reply:
It can't be fixed for consumers, since components are not implemented in generators side.

Reply:
I'll leave this unaddressed.
I think the future component support of the cmake* generators will keep the current behavior working.
@danimtb Am I correct?

Reply:
Yes, Components will be able to fix it.

Reply:
Components integration with the `cmake_find_package` generator will be released in `1.26` (probably next week)

Reply:
@madebr Now you can keep the older version, as OpenSSL offers the same targets as expected originally.

Reply:
Removed it for the 1.10.x releases.